name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/dkb/DkbErtragsgutschriftDividende27kstg.txt

```diff
@@ -0,0 +1,30 @@
+10919 Berlin
```

Comment:
Die Testdatei ist hilfreich. Aber ich glaube in Deinem Commit fehlt der eigentlich Test. Kann das sein, dass Du die Änderungen in https://github.com/buchen/portfolio/blob/master/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/dkb/DkbPDFExtractorTest.java nicht dem Commit hinzugefügt hast?